Recommendation.java

```diff
@@ -53,5 +61,9 @@ public void setImgLink(String _imgLink){
     imgLink = _imgLink;
   }
 
+  public void changeDislikes(int newDislikes) {
```

Comment:
please change this to be addDislike() with no parameters
then just increase the dislikes by 1